core/src/main/resources/reference.conf

```diff
@@ -466,23 +466,25 @@ cassandra-plugin {
     max-load-attempts = 3
   }
   //#snapshot
-}
 
-# Default dispatcher for plugin actor and tasks.
-cassandra-plugin-default-dispatcher {
-  type = Dispatcher
-  executor = "fork-join-executor"
-  fork-join-executor {
-    parallelism-min = 6
-    parallelism-factor = 1
-    parallelism-max = 6
+  # Default dispatcher for plugin actor and tasks.
+  default-dispatcher {
+    type = Dispatcher
+    executor = "fork-join-executor"
+    fork-join-executor {
+      parallelism-min = 6
+      parallelism-factor = 1
+      parallelism-max = 6
+    }
   }
 }
 
+
+
 //#profile
 datastax-java-driver {
   profiles {
-    cassandra-plugin {
+    akka-persistence-cassandra {
```

Comment:
This difference alone makes this change worth it.
I wonder if adding `-profile` here would make it even more clear.

Reply:
I don't mind but it's somewhat already in scope via the `profiles {` and used in `read-profile =`

Reply:
Here it is clear, but not immediately obvious from where it is referenced in `akka.persistence.cassandra` scope. The `.` vs `-` difference is quite subtile.

Reply:
ok, changed